docs/installation_quickstart.rst

```diff
@@ -62,6 +62,10 @@ your desired settings in place of the example environment variables)::
 For more advanced usage of ``ddtrace-run`` refer to the documentation
 :ref:`here<ddtracerun>`.
 
+Once your application is running you can run the command ``ddtrace-run --info``. 
+This will print out info useful for debugging to make sure the tracer's running as expected and able to connect to the Datadog agent properly. 
```

Comment:
> make sure the tracer's running as expected

this makes it sound like it'll report the health of an existing running tracer, not "validate your env variables"